_includes/card-operation-paid.md

```diff
@@ -0,0 +1,94 @@
+### Card operation `paid-payment` 
```

Comment:
I don't think we need to limit this to Card. We can just call it "Paid Payment" and describe what information the operation gives.